udata/api/__init__.py

```diff
@@ -34,6 +34,23 @@
 
 DEFAULT_PAGE_SIZE = 50
 HEADER_API_KEY = 'X-API-KEY'
+DESCRIPTION = None
+
+FACETS_TYPES = {
+    search.fields.BoolFacet: inputs.boolean
+}
+
+
+def description():
+    '''Load description from markdown file'''
+    if 'short' in request.args:
+        return ''
+    global DESCRIPTION
```

Comment:
Not meant to be in the PR. I'll extract it. It was an experiment having as goal:
- all the documentation in the swagger (meant for it)
- description into a single templatable and translatable markdown file (one by langue, we translate the whole file)
- description serialized only when needed (not in admin as example)